pkgs/development/libraries/sentry-native/default.nix

```diff
@@ -0,0 +1,26 @@
+{ lib, stdenv, fetchFromGitHub, cmake, curl, breakpad }:
+
+stdenv.mkDerivation rec {
+  pname = "sentry-native";
+  version = "0.4.11";
+
+  src = fetchFromGitHub {
+    owner = "getsentry";
+    repo = "sentry-native";
+    rev = "${version}";
+    sha256 = "sha256-RmCFpqm82lTh1AHxfTQRsnxTGOw3JowECiLo3pc0/rE=";
+    fetchSubmodules = true;
```

Comment:
Please devendor the libs.

Reply:
Hey, not sure exactly what you mean by this?

Do you mean to get the src files from somewhere else? If so could you point me to an example?

Reply:
We do not want to use the vendored libs. We want to the ones from the package manager to only have one version get updates and security fixes and have nixos specific patches.

Try not fetching the submodules and add the packages to buildInputs if the build script picks them up. If not you can symlink them to the expected locations. If some packages are not in nixpkgs they should be packaged.

Reply:
Ah thanks! Didn't know about this

While digging around in the cmake flags I found one that would use the system libs for breakpad, which is even better I guess!